### PR TITLE
Survey Form Project

### PR DESCRIPTION
**Pull request by shaqayq**
Add files:

- Index.css
- index.html
- Readme.md
- MIT.md

### CODE REVIEW

**Review by shaqayq — COMMENTED** [review]
Hi [@shaqayq]

Good job so far!
There are some issues that you still need to work on!

Required Changes ♻️

-  linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. you can config linter by [](https://github.com/microverseinc/linters-config)

**Comment by shaqayq** [comment]
> Kindly use the class for the semantics tags that you want to apply the style.

File: index.css [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+p {
+  font-size: 1.125rem;
+}
+
+h1,
+p {
+  margin-top: 0;
+  margin-bottom: 0.5rem;
+}
+
```
Comment (shaqayq) [inline index.css:60]: Kindly use the class for the semantics tags that you want to apply the style.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+              <option value="student">Student</option>
+              <option value="job">Full Time Job</option>
+              <option value="learner">Full Time Learner</option>
+              <option value="preferNo">Prefer not to say</option>
+              <option value="other">Other</option>
+            </select>
+          </div>
+      
+          <div class="form-group">
+            <p>Would you recommend freeCodeCamp to a friend?</p>
```
Comment (shaqayq) [inline index.html:69]: I think it is better to use the <h6> tag instead <p> tag to get better attention from the user.

**Review by shaqayq — COMMENTED** [review]
Hi @shaqayq.
Good job so far! 👍🏻
There are some issues that you still need to work on.

**Review by KayLemba — APPROVED** [review]
Hi @shaqayq 😃, 

You did a great job delivering such an amazing review 💯 Great summary 👍 Well done using the GitHub inline comments feature 👍 Polite and well-explained comments 👍

Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**